.all-contributorsrc

```diff
@@ -567,6 +567,15 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
+      "login": "ilai-deutel",
+      "name": "Ilaï Deutel",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/10098207?v=4",
+      "profile": "https://github.com/ilai-deutel",
+      "contributions": [
+        "doc"
+      ]
```

Comment:
@ilai-deutel Would you be okay if I added :package: (for your `platform` contribution)?